app/public/views/periods/PeriodsModel.js

```diff
@@ -178,10 +193,23 @@ export default class PeriodsModel extends Observable {
         return this._hideSelectedPeriods;
     }
 
+    get hideSelectedRows() {
+        return this._hideSelectedPeriods;
+    }
+
```

Comment:
I think better name would be `selectedRowsHidden`, because `hide...` rather suggests doing some operation

Reply:
I agree, changed to `shouldHideSelectedRows` and `_shouldHideSelectedPeriods`.